src/lib/components/globalhealthbar/GlobalHealthBar.component.js

```diff
@@ -0,0 +1,139 @@
+//@flow
+import React from 'react';
+import VegaChart from '../vegachart/VegaChart.component.js';
+
+type Props = {
+  id: string,
+  alertSegments: {
+    severity: 'critical' | 'warning' | 'none',
+    startSegment: string,
+    endSegment: string,
+    description: string,
+  }[],
+  start: { date: number, month: number, year: number },
+  end: { date: number, month: number, year: number },
+  height?: number,
+  width?: number,
+  theme?: 'light' | 'dark' | 'custom',
+  tooltipPosition?: 'up' | 'down',
+};
+
+function GlobalHealthBar({
+  id,
+  alertSegments,
+  start,
+  end,
+  height = 16,
+  width = 650,
+  theme = 'dark',
+  tooltipPosition = 'up',
+}: Props) {
+  const spec = {
```

Comment:
You may be missing a view entry in your spec to clip the segment between the border radius  (I believe that in the current state if a segment is raised at the beginning or end of the bar the border will be squared). You can refer to https://github.com/scality/core-ui/issues/186#issuecomment-792960267 and more specifically this spec section : 
```
"view": {"cornerRadius": 15},
```
and also the layer should then be clipped :
```
mark: {
          type: 'rect',
          tooltip: true,
          cornerRadius: 0,
          cursor: 'pointer',
          clip: true /// <------- here 
        },
        params: [
          { name: 'highlight', select: { type: 'point', on: 'mouseover' } },
        ],
```